specification/resources/resource-manager/Microsoft.Resources/stable/2020-06-01/resources.json

```diff
@@ -2047,6 +2047,41 @@
         }
       }
     },
+    "/providers/Microsoft.Management/managementGroups/{groupId}/providers/{resourceProviderNamespace}/register": {
+      "post": {
+        "tags": [
+          "Providers"
+        ],
+        "operationId": "ManagementGroup_Register",
```

Comment:
let's name it "Providers_ManagementGroup_Register", or "ManagementGroup_Providers_Register".